src/helpers/responseMessage.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Generic to response message
+ * @function
+ *
+ * @param {string} response
+ * @param {string} statusCode
+ * @param {string} object
+ *
+ * @returns {Void} undefined
+ * @description global response message to users
```

Comment:
Move `@description` after `@function`, like a real function `@returns` should be the last thing in a JSDoc. Also, I'm not even sure our JSDocs still need to have the @description keyword since we already described it above.